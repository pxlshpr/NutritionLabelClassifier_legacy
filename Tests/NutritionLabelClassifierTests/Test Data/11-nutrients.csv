attributeString,value1String,value2String
fat,0g,nil
carbohydrate,20g,nil
saturatedFat,0g,nil
dietaryFibre,1g,nil
transFat,0g,nil
sugar,19g,nil
cholesterol,0mg,nil
protein,14g,nil
sodium,65mg,nil
vitaminA,0%,nil
vitaminC,2%,nil
calcium,20%,nil
iron,0%,nil