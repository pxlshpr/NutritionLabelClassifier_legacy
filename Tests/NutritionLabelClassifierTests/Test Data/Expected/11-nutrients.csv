attributeString,value1String,value2String,double,string
energy,746.0kj,497.0kj,,
gluten,0mg,0mg,,
protein,5.7g,3.8g,,
fat,9.5g,6.3g,,
saturatedFat,6.2g,4.1g,,
carbohydrate,17.4g,11.6g,,
sugar,17.1g,11.4g,,
sodium,59.0mg,39.0mg,,
calcium,213.0mg,142.0mg,,